Assets/Code/Player.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+	public Vector2 inputVec;
+	public float speed;
+		
+	Rigidbody2D rigid;
```

Comment:
```suggestion
	// Rigidbody2D 변수 선언
	Rigidbody2D rigid;
```
Rigidbody2D 변수 선언 주석 추가